apps/web/src/services/dashboards/dashboard-detail/DashboardDetailPage.vue

```diff
@@ -90,6 +90,8 @@ import {
 } from 'vue';
 import { useRouter } from 'vue-router';
 
+import { store } from '@/store';
```

Comment:
issue: In Vue instance, please use 'useStore'

Reply:
oh I'll change it, thanks!